lib/passenger.rb

```diff
@@ -16,6 +17,15 @@ def add_trip(trip)
       @trips << trip
     end
 
+    def net_expenditures
+      total_expenditure = @trips.sum(&:cost)
+      return total_expenditure
```

Comment:
You can directly return `@trips.sum(&:cost)`:

```suggestion
      return @trips.sum(&:cost)
```

(This saves you from having to come up with a variable name.)